hack/containerd.toml

```diff
@@ -0,0 +1,77 @@
+root = "/var/lib/containerd"
+state = "/run/containerd"
+oom_score = 0
+subreaper = true
```

Comment:
Think we should just have this?

Reply:
I take it that flags take precedence over the config?

Reply:
i believe so; the above is generated from `containerd config`

Reply:
cool